### ISSUE TITLE
Get https://registry-1.docker.io/v2/qaprosoft/postgres/manifests/9.6: unauthorized: incorrect username or password

### ISSUE THREAD

**Issue by sspieker-cc** [qaprosoft/zafira#1510]
I had attempted to the start of the Zafira Docker image and this message is the result:
Get https://registry-1.docker.io/v2/qaprosoft/postgres/manifests/9.6: unauthorized: incorrect username or password

This is a windows 10 platform and the commands are being run through PowerShell.

This execution had worked earlier this morning but now the server seems to have been reconfigured.  A new clone was created and attempted with the same result.

**Comment by hursevich** [comment]
Not sure that may be the reason, qaprosoft/postgres docker repo is public: https://cloud.docker.com/u/qaprosoft/repository/docker/qaprosoft/postgres
Probably some temp problems with docker cloud.

**Comment by sspieker-cc** [comment]
There seemed to be an issue with the docker install on my machine.  I had re-installed and things were not working overall even after re-installation.

The end result here was that I needed to run the "docker login" command at the command line, then I was able to perform the docker-compose command on my windows 10 machine.

So this was not a Zafira  issue in the end, I was just confused as to what the problem could have been.
